### PR TITLE
Update detail about organization data sharing between live and sandbox

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by mehicalen — DISMISSED** [review]

**Review by noon-dawg — DISMISSED** [review]

**Review by ninjasimon — DISMISSED** [review]

**Review by dara123 — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

**Review by ikram-patah — DISMISSED** [review]
Suggestion: To remove this note line altogether. Having this gives me the impression that the rest (data that are not organizations) are shared between live & sandbox.

**Comment by noon-dawg** [comment]
> Suggestion: To remove this note line altogether. Having this gives me the impression that the rest (data that are not organizations) are shared between live & sandbox.

I think we kind of need to something here because we have changed how this works.

We could update it to something like: 
`Note: No data is shared between the live and sandbox environments.` ?

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -414,7 +414,7 @@ tags:
       You can have multiple organizations.
       For more information, see [My organizations and websites](https://www.rebilly.com/docs/settings/organizations-and-websites/).
 
-      Note: Organizations are shared between the live and sandbox environments.
+      Note: Organizations are not shared between the live and sandbox environments.
```
Comment (ashkarpetin) [inline openapi/openapi.yaml:417]:
```suggestion
      Note: No data is shared between the live and sandbox environments.
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -414,7 +414,7 @@ tags:
       You can have multiple organizations.
       For more information, see [My organizations and websites](https://www.rebilly.com/docs/settings/organizations-and-websites/).
 
-      Note: Organizations are shared between the live and sandbox environments.
+      Note: No data is shared between the live and sandbox environments.
```
Comment (ashkarpetin) [inline openapi/openapi.yaml:417]:
```suggestion
      Note: No data, including organizations, is shared between the live and sandbox environments.
```

**Review by noon-dawg — APPROVED** [review]

**Comment by ashkarpetin** [comment]
Thanks for the feedback.

**Review by mikerebilly — APPROVED** [review]

**Review by lexerom — APPROVED** [review]

**Review by ikram-patah — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Pull request merged**